Examples/SwiftPM/LoggieExample/AppDelegate.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  AppDelegate.swift
+//  iOSExample
+//
+//  Created by Mario Galijot on 23/10/2020.
+//
+
+import UIKit
+
+@main
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
```

Comment:
@nikolamajcen for the warnings, those are resolved a while ago. Please check again on this branch when building the project if those are shown to you.
I did the merging manually since there were some parts missing from master and changes were made on this branch in same places that the SwiftUI support was implemented.